_posts/2021-01-02-Build-the-future-podcast.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Podcast with Build The Future
+banner : images/banner-pulse-standard.jpg
+date:  2021-01-02
+categories: wnc news
+thumbnail: images/pulse-thumbnail-build-the-future.jpg
+---
+
+Our CEO, Mr Manu Ignatius, talked to Cameron Wiese, from Build The Future, about the [future of underwater wireless communications](https://open.spotify.com/episode/1FrGPojKUVl8R9GnVeNiCT).
```

Comment:
```suggestion
Our CEO, Mr Manu Ignatius, talked to Cameron Wiese, from [Build The Future](https://www.buildthefuturepodcast.com/), about the [future of underwater wireless communications](https://www.buildthefuturepodcast.com/episode/manu-ignatius-subnero).
```

Reply:
I would avoid linking to an opaque URL from spotify.

Reply:
What is an opaque URL?
Got it, not URL from Spotify in future.

Reply:
Opaque in the sense this URL https://open.spotify.com/episode/1FrGPojKUVl8R9GnVeNiCT doesn't show which podcast/episode it links to. And there is no guarantee that spotify will link the same podcast from this URL forever.

Reply:
I understand, I'll be more careful in future. Thanks.